src/MspFcComms.hpp

```diff
@@ -74,6 +77,8 @@ namespace FcComms
         // State of communication with flight controller
         FcCommsStatus fc_comms_status_ = FcCommsStatus::kDisconnected;
 
+        uint16_t rc_values[18]{0};
```

Comment:
This variable doesn't need to be global. It's should be passed in by the callee of getRawRc